extensions/amp-story/1.0/story-analytics.js

```diff
@@ -21,7 +21,10 @@ import {map} from '../../../src/utils/object';
 import {registerServiceBuilder} from '../../../src/service';
 import {triggerAnalyticsEvent} from '../../../src/analytics';
 
-/** @private @const {string} */
+/**
+ * @protected
+ * @const {string}
+ **/
```

Comment:
Super nitty: Could we keep this and all the other ones on one line? It doesn't really bring any more clarity and add lines to already really large / hard to browse files

Reply:
Agreed. Done.